Opencv-VS/main.cpp

```diff
@@ -74,7 +74,7 @@ int main(int argc, char* argv[]) {
     cv::imshow(NAME, removedImage);
     cv::waitKey(0);
 
-    cv::Mat binarizedImage = binarizeImageOtsu(removedImage);
+    cv::Mat binarizedImage = binarizeImage(removedImage);
```

Comment:
in my opinion `binarize` is enough because of the `Image` world at the end holds no additional information